portal-impl/src/com/liferay/portal/verify/VerifyDocumentLibrary.java

```diff
@@ -369,12 +370,9 @@ public void performAction(Object object) {
 							dlFileEntry.getFileEntryId(),
 							dlFileEntry.getFileName(), dlFileEntry.getTitle());
 					}
-					catch (PortalException pe) {
-						if (!(pe instanceof DuplicateFileException) &&
-							!(pe instanceof DuplicateFolderNameException)) {
-
-							return;
-						}
+					catch (DuplicateFileException |
```

Comment:
@adolfopa This DuplicateFileException can be removed, we only want to rename title when DuplicateFileEntryException is thrown inside validateFile